## no-rview-please.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: rview was renamed to rvu - seems some stuff still relies on rview...

@DPATCH@
--- a/obj/cabin/anim1/fmt
+++ b/obj/cabin/anim1/fmt
@@ -1 +1 @@
-rview -vtv -vp ${px} ${py} ${pz} -vd ${dx} ${dy} ${dz} -vu ${ux} ${uy} ${uz} $(stuff)
+rvu -vtv -vp ${px} ${py} ${pz} -vd ${dx} ${dy} ${dz} -vu ${ux} ${uy} ${uz} $(stuff)
--- a/obj/cabin/anim1/join.keys
+++ b/obj/cabin/anim1/join.keys
@@ -1,2 +1,2 @@
-rview -vtv -vp 21.1 11.97 4.2 -vd 0.971309 0.0625525 -0.229448 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2.5
-rview -vtv -vp 23.9 12.2 6.2 -vd 0.757833 -0.312891 -0.572528 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 21.1 11.97 4.2 -vd 0.971309 0.0625525 -0.229448 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2.5
+rvu -vtv -vp 23.9 12.2 6.2 -vd 0.757833 -0.312891 -0.572528 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
--- a/obj/cabin/anim1/keys
+++ b/obj/cabin/anim1/keys
@@ -1,55 +1,55 @@
-rview -vtv -vp 4.7 12 4 -vd -0.924626 0.380866 0.00280397 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 0 -e 86
-rview -vtv -vp 4.7 12 4 -vd -0.730485 0.489401 -0.476317 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
-rview -vtv -vp 3.3 13.76 4.2 -vd -0.0427227 0.940831 -0.336174 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .75 -e 55
-rview -vtv -vp 4.3 15.8 4.3 -vd 0.00622979 0.647051 -0.762421 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .75 -e 35
+rvu -vtv -vp 4.7 12 4 -vd -0.924626 0.380866 0.00280397 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 0 -e 86
+rvu -vtv -vp 4.7 12 4 -vd -0.730485 0.489401 -0.476317 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
+rvu -vtv -vp 3.3 13.76 4.2 -vd -0.0427227 0.940831 -0.336174 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .75 -e 55
+rvu -vtv -vp 4.3 15.8 4.3 -vd 0.00622979 0.647051 -0.762421 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .75 -e 35
 pause 2 seconds
-rview -vtv -vp 5.46 15.5 4.2 -vd 0.91674 0.347628 -0.196833 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1 -e 120
-rview -vtv -vp 10.5 15.6 4.2 -vd 0.9896 -0.1408 -0.0294619 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 20.4 13.7 4.2 -vd 0.680414 -0.680414 -0.272166 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 4
+rvu -vtv -vp 5.46 15.5 4.2 -vd 0.91674 0.347628 -0.196833 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1 -e 120
+rvu -vtv -vp 10.5 15.6 4.2 -vd 0.9896 -0.1408 -0.0294619 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 20.4 13.7 4.2 -vd 0.680414 -0.680414 -0.272166 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 4
 pause 1 second
-rview -vtv -vp 22.9 12.2 6 -vd 0.609208 -0.609208 -0.507673 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 24 11.86 5.2 -vd 0.505147 -0.67245 -0.54096 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 24.86 11.2 6 -vd 0.274729 -0.821973 -0.498883 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 25.5 9.5 5.45 -vd -0.242521 -0.882347 -0.403295 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
-rview -vtv -vp 25.4 6.8 5.45 -vd -0.5617 -0.785631 -0.259378 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 25.5 4.55 5.45 -vd -0.736555 -0.376866 -0.561657 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 24.5 2.75 4.2 -vd -0.799281 -0.163636 -0.57825 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 23.1 2.34 6.2 -vd -0.875962 -0.0409728 -0.480637 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 22.2 2.2 5.4 -vd -0.931792 0.318282 -0.17453 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2 -e 100
-rview -vtv -vp 18.4 2 5.4 -vd -0.768929 0.401349 -0.497662 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 16.5 2.5 6.35 -vd -0.273316 0.573355 -0.772375 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 16.25 3.86 6.25 -vd -0.101489 0.720453 -0.686038 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1 -e 120
-rview -vtv -vp 16.35 4.66 5.4 -vd -0.0488706 0.86004 -0.507881 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 16.1 6.4 5.4 -vd 0.241471 0.949711 -0.199351 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 16.6 8.25 5.3 -vd 0.657358 0.694674 -0.292077 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 17.7 9 4.2 -vd 0.829416 0.508562 -0.231157 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 22.9 12.2 6 -vd 0.609208 -0.609208 -0.507673 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 24 11.86 5.2 -vd 0.505147 -0.67245 -0.54096 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 24.86 11.2 6 -vd 0.274729 -0.821973 -0.498883 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 25.5 9.5 5.45 -vd -0.242521 -0.882347 -0.403295 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
+rvu -vtv -vp 25.4 6.8 5.45 -vd -0.5617 -0.785631 -0.259378 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 25.5 4.55 5.45 -vd -0.736555 -0.376866 -0.561657 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 24.5 2.75 4.2 -vd -0.799281 -0.163636 -0.57825 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 23.1 2.34 6.2 -vd -0.875962 -0.0409728 -0.480637 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 22.2 2.2 5.4 -vd -0.931792 0.318282 -0.17453 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2 -e 100
+rvu -vtv -vp 18.4 2 5.4 -vd -0.768929 0.401349 -0.497662 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 16.5 2.5 6.35 -vd -0.273316 0.573355 -0.772375 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 16.25 3.86 6.25 -vd -0.101489 0.720453 -0.686038 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1 -e 120
+rvu -vtv -vp 16.35 4.66 5.4 -vd -0.0488706 0.86004 -0.507881 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 16.1 6.4 5.4 -vd 0.241471 0.949711 -0.199351 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 16.6 8.25 5.3 -vd 0.657358 0.694674 -0.292077 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 17.7 9 4.2 -vd 0.829416 0.508562 -0.231157 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
 repeat previous circle around furniture
-rview -vtv -vp 18.2 11.2 4.2 -vd -0.663625 0.733674 -0.146031 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 14.8 11.1 4.2 -vd -0.845272 -0.514322 -0.144873 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 12.1 8.66 4.2 -vd -0.845272 -0.514322 -0.144873 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 8.25 4 4.2 -vd -0.997702 0.0138135 -0.0663361 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1.5
-rview -vtv -vp 5.3 3.75 5.7 -vd -0.972188 0.206184 -0.111081 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
-rview -vtv -vp 4.6 3.5 6.05 -vd -0.927954 0.356177 -0.109723 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.7 5.6 -vd -0.827968 0.514747 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.5 3.7 6 -vd -0.666529 0.711501 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.55 3.8 5.7 -vd -0.429382 0.875286 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.5 3.75 6.1 -vd 0.0352089 0.942608 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.7 5.6 -vd 0.277974 0.901377 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.65 3.6 6.1 -vd 0.48778 0.807354 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.5 5.6 -vd 0.691421 0.641628 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.65 3.55 6.1 -vd 0.848803 0.41144 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.7 3.6 5.6 -vd 0.934402 0.129008 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.75 3.7 6.1 -vd 0.928534 -0.166052 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.7 3.7 5.7 -vd 0.823885 -0.459307 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.65 3.65 6 -vd 0.641628 -0.691421 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.65 5.7 -vd 0.381566 -0.862645 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.7 6.1 -vd 0.0141714 -0.943159 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.75 5.6 -vd -0.400716 -0.853918 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.55 3.75 6.1 -vd -0.77399 -0.539156 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.5 3.7 5.6 -vd -0.911715 -0.241921 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.5 3.6 6.1 -vd -0.942265 0.0434335 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.7 5.6 -vd -0.827962 0.514976 -0.22199 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
-rview -vtv -vp 3.6 3.7 5.8 -vd -0.827962 0.514976 -0.22199 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
-rview -vtv -vp 3 3.8 4 -vd -0.324443 0.811107 -0.486664 -vu -0.707107 0 0.707107 -vh 60 -vv 49 -vs 0 -vl 0 -t .8
-rview -vtv -vp 2 3.9 3.1 -vd 0.0569963 0.969582 -0.238038 -vu -0.957826 0 0.287348 -vh 60 -vv 49 -vs 0 -vl 0 -t .6
-rview -vtv -vp 1.1 4 2.6 -vd 0.329848 0.943706 0.0248832 -vu -0.998752 0 0.0499376 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
+rvu -vtv -vp 18.2 11.2 4.2 -vd -0.663625 0.733674 -0.146031 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 14.8 11.1 4.2 -vd -0.845272 -0.514322 -0.144873 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 12.1 8.66 4.2 -vd -0.845272 -0.514322 -0.144873 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 8.25 4 4.2 -vd -0.997702 0.0138135 -0.0663361 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1.5
+rvu -vtv -vp 5.3 3.75 5.7 -vd -0.972188 0.206184 -0.111081 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 1
+rvu -vtv -vp 4.6 3.5 6.05 -vd -0.927954 0.356177 -0.109723 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.7 5.6 -vd -0.827968 0.514747 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.5 3.7 6 -vd -0.666529 0.711501 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.55 3.8 5.7 -vd -0.429382 0.875286 -0.222497 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.5 3.75 6.1 -vd 0.0352089 0.942608 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.7 5.6 -vd 0.277974 0.901377 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.65 3.6 6.1 -vd 0.48778 0.807354 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.5 5.6 -vd 0.691421 0.641628 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.65 3.55 6.1 -vd 0.848803 0.41144 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.7 3.6 5.6 -vd 0.934402 0.129008 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.75 3.7 6.1 -vd 0.928534 -0.166052 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.7 3.7 5.7 -vd 0.823885 -0.459307 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.65 3.65 6 -vd 0.641628 -0.691421 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.65 5.7 -vd 0.381566 -0.862645 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.7 6.1 -vd 0.0141714 -0.943159 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.75 5.6 -vd -0.400716 -0.853918 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.55 3.75 6.1 -vd -0.77399 -0.539156 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.5 3.7 5.6 -vd -0.911715 -0.241921 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.5 3.6 6.1 -vd -0.942265 0.0434335 -0.33204 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.7 5.6 -vd -0.827962 0.514976 -0.22199 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t .35
+rvu -vtv -vp 3.6 3.7 5.8 -vd -0.827962 0.514976 -0.22199 -vu 0 0 1 -vh 60 -vv 49 -vs 0 -vl 0 -t 2
+rvu -vtv -vp 3 3.8 4 -vd -0.324443 0.811107 -0.486664 -vu -0.707107 0 0.707107 -vh 60 -vv 49 -vs 0 -vl 0 -t .8
+rvu -vtv -vp 2 3.9 3.1 -vd 0.0569963 0.969582 -0.238038 -vu -0.957826 0 0.287348 -vh 60 -vv 49 -vs 0 -vl 0 -t .6
+rvu -vtv -vp 1.1 4 2.6 -vd 0.329848 0.943706 0.0248832 -vu -0.998752 0 0.0499376 -vh 60 -vv 49 -vs 0 -vl 0 -t .5
--- a/obj/cabin/vf/aerial
+++ b/obj/cabin/vf/aerial
@@ -1 +1 @@
-rview -vtv -vp 64.825 -90.7048 81.4822 -vd -0.399565 0.692276 -0.600917 -vu 0 0 1 -vh 30 -vv 25 -vs 0 -vl 0
+rvu -vtv -vp 64.825 -90.7048 81.4822 -vd -0.399565 0.692276 -0.600917 -vu 0 0 1 -vh 30 -vv 25 -vs 0 -vl 0
--- a/obj/cabin/vf/bath
+++ b/obj/cabin/vf/bath
@@ -1,3 +1,3 @@
-rview -vtv -vp 8 14.5 5 -vd -0.894427 0.447214 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
-rview -vtv -vp 8 14.5 5 -vd -0.874456 0.309256 -0.373747 -vu 0 0 1 -vh 60 -vv 60 -vs 0 -vl 0
-rview -vtv -vp 8 14.5 5 -vd -0.868092 0.4275 -0.252309 -vu 0 0 1 -vh 45 -vv 60 -vo 0 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 8 14.5 5 -vd -0.894427 0.447214 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 8 14.5 5 -vd -0.874456 0.309256 -0.373747 -vu 0 0 1 -vh 60 -vv 60 -vs 0 -vl 0
+rvu -vtv -vp 8 14.5 5 -vd -0.868092 0.4275 -0.252309 -vu 0 0 1 -vh 45 -vv 60 -vo 0 -va 0 -vs 0 -vl 0
--- a/obj/cabin/vf/bed
+++ b/obj/cabin/vf/bed
@@ -1 +1 @@
-rview -vtv -vp 9 12 5 -vd -0.506826 -0.822516 -0.258059 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 9 12 5 -vd -0.506826 -0.822516 -0.258059 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/cabin/vf/bed.leepl
+++ b/obj/cabin/vf/bed.leepl
@@ -1 +1 @@
-rview -vth -vp 7.66807 9.29035 5.55989 -vd -0.583978 -0.790403 -0.185019 -vu 0 0 1 -vh 120 -vv 120 -vs 0 -vl 0
+rvu -vth -vp 7.66807 9.29035 5.55989 -vd -0.583978 -0.790403 -0.185019 -vu 0 0 1 -vh 120 -vv 120 -vs 0 -vl 0
--- a/obj/cabin/vf/bed.leepr
+++ b/obj/cabin/vf/bed.leepr
@@ -1 +1 @@
-rview -vth -vp 7.5034 9.412 5.55989 -vd -0.583978 -0.790403 -0.185019 -vu 0 0 1 -vh 120 -vv 120 -vs 0 -vl 0
+rvu -vth -vp 7.5034 9.412 5.55989 -vd -0.583978 -0.790403 -0.185019 -vu 0 0 1 -vh 120 -vv 120 -vs 0 -vl 0
--- a/obj/cabin/vf/bedroom
+++ b/obj/cabin/vf/bedroom
@@ -1,2 +1,2 @@
-rview -vtv -vp 1.20683 2.12314 3.95566 -vd 0.781959 0.61206 -0.117997 -vu 0 0 1 -vh 40 -vv 30 -vs 0 -vl 0
-rview -vtv -vp 0.5 0.5 5 -vd 0.638787 0.751662 -0.164179 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 1.20683 2.12314 3.95566 -vd 0.781959 0.61206 -0.117997 -vu 0 0 1 -vh 40 -vv 30 -vs 0 -vl 0
+rvu -vtv -vp 0.5 0.5 5 -vd 0.638787 0.751662 -0.164179 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/cabin/vf/coatrack
+++ b/obj/cabin/vf/coatrack
@@ -1 +1 @@
-rview -vtv -vp 22 12 5 -vd 0.563075 0.755671 -0.334526 -vu 0 0 1 -vh 40 -vv 55 -vs 0 -vl 0
+rvu -vtv -vp 22 12 5 -vd 0.563075 0.755671 -0.334526 -vu 0 0 1 -vh 40 -vv 55 -vs 0 -vl 0
--- a/obj/cabin/vf/corner
+++ b/obj/cabin/vf/corner
@@ -1,3 +1,3 @@
-rview -vtv -vp 22.4986 3.54532 3.96568 -vd -0.910534 -0.363276 -0.19738 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
-rview -vtv -vp 22.4986 3.54532 3.96568 -vd -0.919145 -0.363618 -0.151507 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
-rview -vtv -vp 22.4986 3.54532 3.96568 -vd -0.929463 -0.3677 -0.0299152 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 22.4986 3.54532 3.96568 -vd -0.910534 -0.363276 -0.19738 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 22.4986 3.54532 3.96568 -vd -0.919145 -0.363618 -0.151507 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 22.4986 3.54532 3.96568 -vd -0.929463 -0.3677 -0.0299152 -vu 0 0 1 -vh 51.7865 -vv 45 -vo 0 -va 0 -vs 0 -vl 0
--- a/obj/cabin/vf/ext.ne
+++ b/obj/cabin/vf/ext.ne
@@ -1 +1 @@
-rview -vtv -vp 58.0405 72.6078 41.9572 -vd -0.521721 -0.721022 -0.455998 -vu 0 0 1 -vh 27.7007 -vv 27.7007 -vs 0 -vl 0
+rvu -vtv -vp 58.0405 72.6078 41.9572 -vd -0.521721 -0.721022 -0.455998 -vu 0 0 1 -vh 27.7007 -vv 27.7007 -vs 0 -vl 0
--- a/obj/cabin/vf/ext.se
+++ b/obj/cabin/vf/ext.se
@@ -1 +1 @@
-rview -vtv -vp 47.9362 -54.2003 40.135 -vd -0.445657 0.772134 -0.452989 -vu 0 0 1 -vh 30 -vv 25 -vs 0 -vl 0
+rvu -vtv -vp 47.9362 -54.2003 40.135 -vd -0.445657 0.772134 -0.452989 -vu 0 0 1 -vh 30 -vv 25 -vs 0 -vl 0
--- a/obj/cabin/vf/fireplace
+++ b/obj/cabin/vf/fireplace
@@ -1 +1 @@
-rview -vtv -vp 12.867 10.3625 4.60554 -vd 0.958111 -0.223918 -0.178561 -vu 0 0 1 -vh 23.6949 -vv 23.6949 -vs 0 -vl 0
+rvu -vtv -vp 12.867 10.3625 4.60554 -vd 0.958111 -0.223918 -0.178561 -vu 0 0 1 -vh 23.6949 -vv 23.6949 -vs 0 -vl 0
--- a/obj/cabin/vf/interior
+++ b/obj/cabin/vf/interior
@@ -1 +1 @@
-rview -vtv -vp 26 9.4 5 -vd -0.952099 0.304778 -0.0248474 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 26 9.4 5 -vd -0.952099 0.304778 -0.0248474 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/cabin/vf/liv3
+++ b/obj/cabin/vf/liv3
@@ -1,2 +1,2 @@
-rview -vtv -vp 26 10 3.5 -vd -1 0 0 -vu 0 0 1 -vh 60 -vv 60 -vo 0 -va 0 -vs 0 -vl 0
-rview -vtv -vp 26 10 3.5 -vd -1 0 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 26 10 3.5 -vd -1 0 0 -vu 0 0 1 -vh 60 -vv 60 -vo 0 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 26 10 3.5 -vd -1 0 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
--- a/obj/cabin/vf/liv4
+++ b/obj/cabin/vf/liv4
@@ -1 +1 @@
-rview -vtv -vp 25.7995 12.8471 3.34734 -vd -0.984808 -0.173648 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 25.7995 12.8471 3.34734 -vd -0.984808 -0.173648 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
--- a/obj/cabin/vf/liv7
+++ b/obj/cabin/vf/liv7
@@ -1 +1 @@
-rview -vtv -vp 25.1293 15.4706 3.31179 -vd -0.939693 -0.34202 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
+rvu -vtv -vp 25.1293 15.4706 3.31179 -vd -0.939693 -0.34202 0 -vu 0 0 1 -vh 60 -vv 60 -vo 4 -va 0 -vs 0 -vl 0
--- a/obj/cabin/vf/living
+++ b/obj/cabin/vf/living
@@ -1 +1 @@
-rview -vtv -vp 12.5 0.5 5 -vd 0.707107 0.707107 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 12.5 0.5 5 -vd 0.707107 0.707107 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/cabin/vf/living.fish
+++ b/obj/cabin/vf/living.fish
@@ -1 +1 @@
-rview -vta -vp 15 9 5 -vd 1 0 0 -vu 0 0 1 -vh 240 -vv 180 -vs 0 -vl 0
+rvu -vta -vp 15 9 5 -vd 1 0 0 -vu 0 0 1 -vh 240 -vv 180 -vs 0 -vl 0
--- a/obj/cabin/vf/living2
+++ b/obj/cabin/vf/living2
@@ -1,3 +1,3 @@
-rview -vtv -vp 26.5 17.5 5 -vd -0.980581 -0.196116 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
-rview -vtv -vp 26.5 17.5 5 -vd -0.662847 -0.740421 -0.111404 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
-rview -vtv -vp 26.8 16.8 5 -vd -0.662847 -0.740421 -0.111404 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 26.5 17.5 5 -vd -0.980581 -0.196116 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 26.5 17.5 5 -vd -0.662847 -0.740421 -0.111404 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 26.8 16.8 5 -vd -0.662847 -0.740421 -0.111404 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/cabin/vf/plan
+++ b/obj/cabin/vf/plan
@@ -1 +1 @@
-rview -vtl -vp 13.5 9 4 -vd 0 0 -1 -vu 0 1 0 -vh 27 -vv 18 -vs 0 -vl 0
+rvu -vtl -vp 13.5 9 4 -vd 0 0 -1 -vu 0 1 0 -vh 27 -vv 18 -vs 0 -vl 0
--- a/obj/cabin/vf/top
+++ b/obj/cabin/vf/top
@@ -1,2 +1,2 @@
-rview -vtv -vp 14.3846 -55.8279 63.7164 -vd 0 0.707107 -0.707107 -vu 0 0 1 -vh 45 -vv 45 -vs 0 -vl 0
-rview -vtv -vp 62.9669 -16.2963 53.0042 -vd -0.632252 0.367724 -0.681936 -vu 0 0 1 -vh 32.9635 -vv 32.9635 -vs 0 -vl 0
+rvu -vtv -vp 14.3846 -55.8279 63.7164 -vd 0 0.707107 -0.707107 -vu 0 0 1 -vh 45 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 62.9669 -16.2963 53.0042 -vd -0.632252 0.367724 -0.681936 -vu 0 0 1 -vh 32.9635 -vv 32.9635 -vs 0 -vl 0
--- a/obj/misc/Makefile
+++ b/obj/misc/Makefile
@@ -5,7 +5,7 @@
 DEV=x11
 
 daf:	daf.oct
-	rview -o $(DEV) -vp 50 60 40 -vd 0 -0.707107 -0.707107 -vu 0 1 0 \
+	rvu -o $(DEV) -vp 50 60 40 -vd 0 -0.707107 -0.707107 -vu 0 1 0 \
 -vh 20 -vv 20 -av .2 .2 .2 daf.oct
 
 daffodil.hdr:	daf.oct
@@ -19,7 +19,7 @@ daffodil.xf:	daffodil
 	xform -e -rz -40 -t 50 40 20 daffodil > daffodil.xf
 
 exm:	examp.oct
-	rview -o $(DEV) -vp 3 2 5 -vd -0.268328 -0.357771 -0.894427 \
+	rvu -o $(DEV) -vp 3 2 5 -vd -0.268328 -0.357771 -0.894427 \
 -vu 0 1 0 -vh 45 -vv 45 -av .25 .25 .25 examp.oct
 
 examp.hdr:	examp.oct
--- a/obj/misc/daffodil.vp
+++ b/obj/misc/daffodil.vp
@@ -1 +1 @@
-rview -vp 50 60 40 -vd 0 -0.707107 -0.707107 -vu 0 1 0 -vh 20 -vv 20 -x 470 -y 470
+rvu -vp 50 60 40 -vd 0 -0.707107 -0.707107 -vu 0 1 0 -vh 20 -vv 20 -x 470 -y 470
--- a/obj/misc/examp.vp
+++ b/obj/misc/examp.vp
@@ -1 +1 @@
-rview -vp 3 2 5 -vd -0.268328 -0.357771 -0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 470 -y 470
+rvu -vp 3 2 5 -vd -0.268328 -0.357771 -0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 470 -y 470
--- a/obj/office/Makefile
+++ b/obj/office/Makefile
@@ -6,7 +6,7 @@ VIEW = -vp 8 36 -27 -vd -.46 -.23 .79 -v
 DEV = x11
 
 view1:	model.oct
-	rview $(VIEW) -o $(DEV) -av .1 .1 .1 model.oct
+	rvu $(VIEW) -o $(DEV) -av .1 .1 .1 model.oct
 
 display:	/usr2/greg/pic/model.new
 	ximage =+50+50 -f /usr2/greg/pic/model.new
--- a/obj/office/book.vp
+++ b/obj/office/book.vp
@@ -1,3 +1,3 @@
-rview -vp -25 36 25 -vd 0.0885656 -0.437046 -0.895068 -vu 0 1 0 -vh 16.5224 -vv 11.1805 -x 1000 -y 676
-rview -vp -13.8209 31.2191 -2.57023 -vd -0.761583 -0.55009 0.342625 -vu 0 1 0 -vh 30.8742 -vv 30.8742 -x 800 -y 800
-rview -vp -20 33 -3.5 -vd -0.254777 -0.832881 0.491322 -vu 0 1 0 -vh 30.8742 -vv 30.8742 -x 800 -y 800
+rvu -vp -25 36 25 -vd 0.0885656 -0.437046 -0.895068 -vu 0 1 0 -vh 16.5224 -vv 11.1805 -x 1000 -y 676
+rvu -vp -13.8209 31.2191 -2.57023 -vd -0.761583 -0.55009 0.342625 -vu 0 1 0 -vh 30.8742 -vv 30.8742 -x 800 -y 800
+rvu -vp -20 33 -3.5 -vd -0.254777 -0.832881 0.491322 -vu 0 1 0 -vh 30.8742 -vv 30.8742 -x 800 -y 800
--- a/obj/office/chair2.vp
+++ b/obj/office/chair2.vp
@@ -1,2 +1,2 @@
-rview -vtv -vp 18.7204 34.1614 -7.45533 -vd -0.946595 -0.224341 0.231581 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
-rview -vtv -vp 18.7 36 -7.5 -vd -0.936836 -0.266259 0.226813 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
+rvu -vtv -vp 18.7204 34.1614 -7.45533 -vd -0.946595 -0.224341 0.231581 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
+rvu -vtv -vp 18.7 36 -7.5 -vd -0.936836 -0.266259 0.226813 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
--- a/obj/office/closeup.vp
+++ b/obj/office/closeup.vp
@@ -1 +1 @@
-rview -vtv -vp -6.94171 28.0803 -12.3719 -vd -0.487983 -0.243991 0.838058 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
+rvu -vtv -vp -6.94171 28.0803 -12.3719 -vd -0.487983 -0.243991 0.838058 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
--- a/obj/office/model.vp
+++ b/obj/office/model.vp
@@ -1,3 +1,3 @@
-rview -vp 0 36 -25 -vd -0.357771 -0.268328 0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 800 -y 800
-rview -vp -3 36 -25 -vd -0.357771 -0.268328 0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 800 -y 800
-rview -vp 8 36 -27 -vd -0.560723 -0.233635 0.794358 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
+rvu -vp 0 36 -25 -vd -0.357771 -0.268328 0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 800 -y 800
+rvu -vp -3 36 -25 -vd -0.357771 -0.268328 0.894427 -vu 0 1 0 -vh 45 -vv 45 -x 800 -y 800
+rvu -vp 8 36 -27 -vd -0.560723 -0.233635 0.794358 -vu 0 1 0 -vh 39.9 -vv 27.5 -x 1000 -y 676
--- a/obj/virtual/back.vp
+++ b/obj/virtual/back.vp
@@ -1,2 +1,2 @@
-rview -vtv -vp 0.1 7.9 2 -vd 0.5547 -0.83205 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
-rview -vtv -vp 0.1 7.9 2 -vd 0.560199 -0.81744 -0.134051 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 0.1 7.9 2 -vd 0.5547 -0.83205 0 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
+rvu -vtv -vp 0.1 7.9 2 -vd 0.560199 -0.81744 -0.134051 -vu 0 0 1 -vh 60 -vv 45 -vs 0 -vl 0
--- a/obj/virtual/ceil.vp
+++ b/obj/virtual/ceil.vp
@@ -1 +1 @@
-rview -vtl -vp 3 4 2.25 -vd 0 0 1 -vu 1 0 0 -vh 8 -vv 6 -vs 0 -vl 0
+rvu -vtl -vp 3 4 2.25 -vd 0 0 1 -vu 1 0 0 -vh 8 -vv 6 -vs 0 -vl 0
--- a/obj/virtual/outside.vp
+++ b/obj/virtual/outside.vp
@@ -1,2 +1,2 @@
 VIEW= -vp -10 -10 5 -vd 1 1 -.5
-rview -vtv -vp -34.3862 -18.695 11.6818 -vd 0.886789 0.382957 -0.258747 -vu 0 0 1 -vh 30 -vv 20 -vs 0 -vl 0
+rvu -vtv -vp -34.3862 -18.695 11.6818 -vd 0.886789 0.382957 -0.258747 -vu 0 0 1 -vh 30 -vv 20 -vs 0 -vl 0
--- a/src/px/pacuity.csh
+++ b/src/px/pacuity.csh
@@ -73,7 +73,7 @@ end
 set pf=( $pf $ifile )
 rcalc -n -e "$vp" -e "A:3879*sqrt(Vhn/$pres[1]*Vvn/$pres[2])" \
 	-o '${60/A});' >> $tfc1
-getinfo < $ifile | egrep '^((VIEW|EXPOSURE|PIXASPECT|PRIMARIES|COLORCORR)=|[^ ]*(rpict|rview|pinterp) )'
+getinfo < $ifile | egrep '^((VIEW|EXPOSURE|PIXASPECT|PRIMARIES|COLORCORR)=|[^ ]*(rpict|rvu|pinterp) )'
 pcomb -f $tfc1 $pf
 quit:
 rm -f $tf
--- a/src/px/psquish.csh
+++ b/src/px/psquish.csh
@@ -191,7 +191,7 @@ _EOF_
 	endif
 endif
 # Map our picture
-getinfo < $ifile | egrep '^((VIEW|PIXASPECT|PRIMARIES)=|[^ ]*(rpict|rview|pinterp) )'
+getinfo < $ifile | egrep '^((VIEW|PIXASPECT|PRIMARIES)=|[^ ]*(rpict|rvu|pinterp) )'
 pcomb -f $tf4 -f $tf3 $ifile
 quit:
 rm -f $tf
--- a/src/px/pveil.csh
+++ b/src/px/pveil.csh
@@ -36,7 +36,7 @@ ro = ri(1) + veil;
 go = gi(1) + veil;
 bo = bi(1) + veil;
 '_EOF_'
-getinfo < $ifile | egrep '^((VIEW|EXPOSURE|PIXASPECT|PRIMARIES|COLORCORR)=|[^ ]*(rpict|rview|pinterp) )'
+getinfo < $ifile | egrep '^((VIEW|EXPOSURE|PIXASPECT|PRIMARIES|COLORCORR)=|[^ ]*(rpict|rvu|pinterp) )'
 pcomb -f $cf $ifile
 quit:
 rm -f $tf
